ledger/src/blockstore.rs

```diff
@@ -9618,6 +9653,7 @@ pub mod tests {
                                 .map(|key| (key, true)),
                             TransactionStatusMeta::default(),
                             counter,
+                            None,
                         )
```

Comment:
We should probably have a unit test that confirms the new batching behavior

Reply:
Added.